### DISCUSSION TITLE
Conditional statements

### DISCUSSION THREAD

**Discussion by borisd** [guidone/lets-form#27] [Ideas]
Any plans to add conditional inputs/areas?

**Comment by guidone** [comment]
Hi @borisd, sorry for anwering late, didn't receive the notification for this.

By design I've avoided to implement convoluted UI for rules like hiding / showing, the LetsForm target are developers, they know how to use javascript, so most of this can be accomplished with [scripts](https://letsform.dev/advanced/form-scripts).
I've to admit that most of the times in scripts, users just want to hide / show elements, so perhaps a UI shortcut could be useful.

One example is here: https://designer.letsform.dev/f/82xk8n76o8c

Another example is here: https://designer.letsform.dev/f/iy9jiwh88n8

(both taken from Templates section of the Designer)

**Reply by borisd** [reply]
@guidone thank you for the reply. We were trying to see if lets-form can be a replacement for JSON-Schema generated forms (with the many downsides that spec has).

Perhaps the scripts option can solve some of the issues.

Thanks for all your work!
